scripts/Dockerfile.serverless-init.alpine.build

```diff
@@ -2,3 +2,4 @@ FROM scratch
 ARG TARGETARCH
 
 COPY ./bin/datadog_extension-$TARGETARCH-alpine/extensions/datadog-agent /datadog-init
+COPY ./bin/serverless_init_dotnet.sh /dotnet.sh
```

Comment:
do we have to add this for some reason for alpine?

Reply:
ah yeah, you're right, that's unnecessary for alpine ✏️